mqtt_subscriber/main.py

```diff
@@ -2,16 +2,22 @@
 import traceback
 from mqtt_subscriber import generate_topic_api_client_args_mapping
 from mqtt_subscriber.oli_broker import MQTTConnection
-
+from multiprocessing import Process
+from websocket_subscriber import create_live_data_consumer
 
 def main():
     logging.getLogger().setLevel(logging.INFO)
     topic_api_client_mapping = generate_topic_api_client_args_mapping()
 
     # Connect to the MQTT broker
-    mqtt_connection = MQTTConnection(topic_api_client_mapping)
     try:
-        mqtt_connection.run_forever()
+        p1 = Process(target=create_live_data_consumer())
+        p1.start()
+
+        p2 = Process(target=MQTTConnection(topic_api_client_mapping).run_forever())
+        p2.start()
```

Comment:
The problem with processes is that if one of the 2 crashes, this will not result in the mqtt_subscriber pod terminating. For instance, think of the case where the websocket process dies, and the mqtt process continues to operate. The pod will not restart, therefore the websocket process will never restart (the pod will keep on running only with the mqtt process). 
On another note, I am not sure it is going to work as expected in dev env, because the main process will terminate (there is no `join()` call for the processes as far as I see)

Reply:
After offline discussion

- No need to have `.join` as python library do so.
- Both `MQTT`  and `websocket`  could be launch as two separate processes via main.py script.